tests/src/test/java/alluxio/client/cli/fs/AbstractShellIntegrationTest.java

```diff
@@ -32,15 +33,14 @@
   @Rule
   public LocalAlluxioClusterResource mLocalAlluxioClusterResource = buildLocalAlluxioCluster();
 
-  protected void customizeLocalAlluxioCluster(LocalAlluxioClusterResource.Builder resource) { }
-
   public LocalAlluxioClusterResource buildLocalAlluxioCluster() {
     LocalAlluxioClusterResource.Builder resource = new LocalAlluxioClusterResource.Builder()
         .setProperty(PropertyKey.WORKER_MEMORY_SIZE, SIZE_BYTES)
         .setProperty(PropertyKey.USER_BLOCK_SIZE_BYTES_DEFAULT, SIZE_BYTES)
         .setProperty(PropertyKey.MASTER_TTL_CHECKER_INTERVAL_MS, Integer.MAX_VALUE)
-        .setProperty(PropertyKey.USER_FILE_WRITE_TYPE_DEFAULT, "CACHE_THROUGH");
-    customizeLocalAlluxioCluster(resource);
+        .setProperty(PropertyKey.USER_FILE_WRITE_TYPE_DEFAULT, "CACHE_THROUGH")
+        .setProperty(PropertyKey.MASTER_RPC_PORT, PortRegistry.acquirePort())
+        .setProperty(PropertyKey.MASTER_WEB_PORT, PortRegistry.acquirePort());
```

Comment:
Setting these here with the `acquirePort` won't actually set the `ServerConfiguration` until we call `start()` on the cluster resource. They should probably be acquired and set beforehand.

Otherwise we can set the configuration once the `build` method is called in between creating the cluster and actually starting the processes.